app/views/nodes/items_table/_table.html.erb

```diff
@@ -2,9 +2,10 @@
   <div class="table-wrapper">
     <table class="table table-striped"
       data-behavior="dradis-datatable"
-      data-default-columns='["Title", "Created", "Updated"]'
-      data-item-name="<%= items.first.class.name.underscore %>"
-      data-local-storage-key="project.ce.<%= dom_id(current_project) %>.<%= items.first.class.name.underscore.pluralize %>_datatable">
+      data-default-columns=<%= @default_columns.fetch(class_name.to_sym, ['Title', 'Created', 'Updated', 'Validation']).to_json %>
```

Comment:
smell

Reply:
Updated to use a hash with default values for simplicity ✅